tests/flow/test_ts_resp3.py

```diff
@@ -60,7 +60,7 @@ def test_resp3(env):
                     b't2{1}': [{b'name': b'mush'}, [990, 10.0]]}
 
         res = r1.execute_command('ts.queryindex', 'name=mush')
-        assert res == {b't1{1}', b't2{1}'}
+        assert res == [b't1{1}', b't2{1}']
```

Comment:
is it correct of us to demand ordering of something that should rightly be unordered? would it not be better to compare sets?

Reply:
A great question! A year or so ago I asked the same someone, and got an answer that we shouldn't, but at the same time we should, as sort of "some random is less of a random than the other random". I, however, don't exactly know if this is related to this particular line or not, but in the end, I don't think we should ever rely on the collections we aren't exactly 100% sure which should return the elements in some particular order like this.

So, for now, it works, and I just didn't come back to it after, and I think this change was required because I downgraded from 0.7.5 to 0.7.2 rltest, so wasn't really paying attention to it and just wanted to silence it.

Reply:
i think it should be `assert set(res) == set([...])`

Reply:
Yes, should be great and simple for the test.